manifest.json

```diff
@@ -46,12 +46,6 @@
           "path": "adobedocs/adobeio-events/master/using/aem-event-setup.md",
           "title": "AEM Events",
           "pages": [
-            {
-              "importedFileName": "aem_on_premise_install_6.3",
-              "path": "adobedocs/adobeio-events/master/aem/aem_on_premise_install_6.3.md",
-              "title": "'AEM 6.2.x & 6.3.x on Premise' Events",
```

Comment:
Even though we stopped supporting AEM 6.3, I am not sure if hiding the pages without any prior notice is the way to do it. We should instead mark the 6.3 page as deprecated and remove it only if there are no active integrations. WDYT?

Reply:
sure... I added the aem 6.3 link back in the main `aem-event-setup.md` page

Reply:
however to keep the left side menu cleaner, I removed the 6.3 links from there.

Reply:
Makes sense.